modules/appeals_api/app/models/appeals_api/higher_level_review.rb

```diff
@@ -132,11 +132,23 @@ def veteran_phone_number
       veteran_phone.to_s
     end
 
+    def veteran_phone_data
+      veteran&.dig('phone')
+    end
+
+    def veteran_homeless?
+      form_data&.dig('data', 'attributes', 'veteran', 'homeless')
+    end
+
     # 11. E-MAIL ADDRESS
     def email
       veteran&.dig('emailAddressText').to_s.strip
     end
 
+    def email_v2
```

Comment:
I'm not sure that I know how to solve this, but I feel like it stinks that we need to do this.

Reply:
Very much agreed. I also played with passing a version into the `email` and found that more distasteful :(